lib/flash.robot

```diff
@@ -21,7 +21,6 @@ Flash Via Internal Programmer With Args
         IF    "Warning: Chip content is identical to the requested image." in """${out_flash}"""
             RETURN
         END
-        Should Contain    ${out_flash}    VERIFIED
```

Comment:
Flashing on V540TND here does not ever say VERIFIED, verification is not performed because the flash device is not fully supported or something